run_preprocessing.py

```diff
@@ -176,17 +151,58 @@ def preprocess(config: PreprocessingConfig):
         .to_numpy()
     )
 
-    if config.save_to_csv:
-        save_to_csv(customers, articles, transactions)
+    per_article_img_embedding = torch.zeros((0, 512))
+    if config.load_image_embedding:
+        print("| Adding image embeddings...")
+        image_embeddings = torch.load(
+            "data/derived/fashion-recommendation-image-embeddings-clip-ViT-B-32.pt"
+        )
+        for index, article in tqdm(articles.iterrows()):
+            per_article_img_embedding = torch.cat(
+                (
+                    per_article_img_embedding,
+                    image_embeddings.get(
+                        int(article["article_id"]), torch.zeros(512)
+                    ).unsqueeze(0),
+                ),
+                axis=0,
+            )
+
+    per_article_text_embedding = torch.zeros((0, 512))
+    if config.load_text_embedding:
+        print("| Adding text embeddings...")
+        text_embeddings = torch.load(
+            "data/derived/fashion-recommendation-text-embeddings-clip-ViT-B-32.pt"
+        )
+
+        # for key in ["derived_name", "derived_look", "derived_category"]:
+        for index, article in tqdm(articles.iterrows()):
+            per_article_text_embedding = torch.cat(
+                (
+                    per_article_text_embedding,
+                    text_embeddings[int(article["article_id"])]
+                    .get("derived_look", torch.zeros(512))
```

Comment:
config?

Reply:
config sounds good, I'll make it happen!